### PR TITLE
Support for config.defaultPageSizes

### PR DESCRIPTION
**Pull request by gor181**
#13 

Usage as:

```
export default sematable('reducerNameSpace', Component, columns, {
  defaultPageSizes: [5, 25, 50, 100, 200, -1],
});
```

### CODE REVIEW

File: src/PageSize.js [hunk 1]
```diff
@@ -32,11 +37,16 @@ class PageSize extends Component {
             margin: '0 0 0 5px',
           }}
         >
-          <option value={5}>5</option>
-          <option value={10}>10</option>
-          <option value={50}>50</option>
-          <option value={100}>100</option>
-          <option value={-1}>All</option>
```
Comment (amir-hadzic) [inline src/PageSize.js]: Can we use PAGE_SIZE_ALL here?

File: src/reducer.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { createTextFilter, createValueFilter } from './common';
 const defaultState = (configs = {}) => ({
   page: 0,
   pageSize: configs.defaultPageSize || 5,
+  pageSizes: configs.defaultPageSizes || [5, 10, 50, 100, -1],
```
Comment (amir-hadzic) [inline src/reducer.js]: Can we use PAGE_SIZE_ALL here?

File: src/PageSize.js (hunk 1 above)
Comment (gor181) [inline src/PageSize.js]: Not sure what you mean, we use `PAGE_SIZE_ALL` at line 46 in order to render the `All` option.

File: src/reducer.js (hunk 2 above)
Comment (gor181) [inline src/reducer.js]: Yap done.

**Comment by gor181** [comment]
Hey @amir-hadzic,

Some improvements made, do let me know if theres anything else left.

Cheers,

File: src/PageSize.js (hunk 1 above)
Comment (amir-hadzic) [inline src/PageSize.js]: Ignore me, these lines were removed :+1:

**Comment by amir-hadzic** [comment]
Thanks @gor181 !

**Pull request merged**